account.go

```diff
@@ -637,6 +637,36 @@ type AccountController struct {
 	Type         AccountControllerType `json:"type"`
 }
 
+// Fields that are due and can be satisfied by providing the corresponding alternative fields instead.
+type AccountFutureRequirementsAlternative struct {
+	AlternativeFieldsDue []string `json:"alternative_fields_due"`
+	OriginalFieldsDue    []string `json:"original_fields_due"`
+}
+
+// Fields that are `currently_due` and need to be collected again because validation or verification failed.
+type AccountFutureRequirementsError struct {
+	Code        string `json:"code"`
+	Reason      string `json:"reason"`
+	Requirement string `json:"requirement"`
+}
+
+type AccountFutureRequirements struct {
+	Alternatives        []*AccountFutureRequirementsAlternative `json:"alternatives"`
+	CurrentDeadline     int64                                   `json:"current_deadline"`
+	CurrentlyDue        []string                                `json:"currently_due"`
+	DisabledReason      string                                  `json:"disabled_reason"`
```

Comment:
This is tricky, it's the same as `AccountRequirementsDisabledReason` right? It should be either that enum or its own enum